app/models/profile_pin.rb

```diff
@@ -1,13 +1,14 @@
-class ProfilePin < ApplicationRecord
+class GenericPins < ApplicationRecord #migrate later
```

Comment:
This won't work, I think you should keep the name for now. You can worry about the model name in a separate PR once the functionality is in.

Reply:
Yup, this makes sense to me.

Reply:
I see, will do.